src/connectors/index.ts

```diff
@@ -70,8 +70,7 @@ export const walletconnect = new WalletConnectConnector({
     [BINANCE_CHAIN_ID]: BINANCE_NETWORK_URL
   },
   bridge: WALLETCONNECT_BRIDGE,
-  qrcode: true,
-  pollingInterval: 15000
+  qrcode: true
```

Comment:
ohhh that's the pollingInterval for WC?

Reply:
yes, the version of @walletconnect/web3-provider we are using doesn't support passing in the value